dspace/config/modules/ldn.cfg

```diff
@@ -45,6 +37,8 @@ ldn.notify.inbox.block-untrusted = true
 # and out-of-range IP
 ldn.notify.inbox.block-untrusted-ip = true
 
+#ldn.notify.relation.metadata = dc.relation
```

Comment:
@steph-ieffam : Please add a description to this metadata field.  All configurations must be described in configuration files, so that others can understand how to use them.

Reply:
@tdonohue I have added the missing comment